rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/component/Sw360ComponentService.java

```diff
@@ -69,6 +69,11 @@ public RequestStatus updateComponent(Component component, User sw360User) throws
         return requestStatus;
     }
 
+    public RequestStatus deleteComponent(String componentId, User sw360User) throws TException {
```

Comment:
Maybe update license header to `Copyright Bosch Software Innovations GmbH, 2018`

Reply:
License header is updated.